app/models/provider.rb

```diff
@@ -156,6 +156,8 @@ def current_accredited_courses
 
   validate :add_enrichment_errors
 
+  validates :accredited_provider_id, accredited_provider_id_format: { allow_blank: false }, on: :update, if: :accredited_provider?
```

Comment:
I think this might introduce an additional problem if providers (not support) try to update some other info and it fails because of this validation. At the moment, this isn't exposed as a field for them to edit so it'll be a dead end. Maybe we allow a blank value but only try to validate if one is provided (in which case it will only trigger from the support side if that makes sense)

Reply:
Yeah it's tricky

Reply:
I think that should work